js/appInit.ts

```diff
@@ -36,13 +36,21 @@ function configureBindings() {
     }
 }
 
+function registerTableView() {
+    window["TableView"] = TableView;
+}
+
+function registerActionBlock() {
```

Comment:
Don't expose this as two functions. One function would be enough. `registerTableView` should be sufficient.